src/components/solverComponents/ProgressHandler.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Spinner } from "@chakra-ui/react";
+import { Container } from "../Container";
+import { GoButton } from "./GoButton";
+
+interface IProgressHandlerProps {
+
+}
```

Comment:
this interface is unnecessary, you can just write `{}` in the `React.Component<{},XyzState>` type parameters

Reply:
did that